test/support/noapi_controller_with_struct_specs.ex

```diff
@@ -0,0 +1,18 @@
+defmodule OpenApiSpexTest.NoApiControllerWithStructSpecs do
+  @moduledoc false
+
+  use Phoenix.Controller
+
+  plug OpenApiSpex.Plug.CastAndValidate, json_render_error_v2: true
+
+  def open_api_operation(action) do
+    apply(__MODULE__, :"#{action}_operation", [])
+  end
+
+  def noapi_operation(), do: nil
```

Comment:
Returning `false` is more explicit, intention revealing compared to `nil` and it's a documented behaviour in the [readme](https://github.com/open-api-spex/open_api_spex#operations). When an operation is nil, it probably means a function expected to return an operation is not working as intended and instead of hiding this error, I suggest guiding the user through a more informative error message to either:

1. Disable cast and validate for multiple actions with:

```elixir
plug OpenApiSpex.Plug.CastAndValidate, json_render_error_v2: true when action not in [:noapi_operation]
```

3. Change `noapi_operation` to return false
4. Change `noapi_operation` to return an `%OpenApiSpex.Operation{}`

Reply:
The reason why I picked `nil` is because this is what the code in `controller_spec` is doing ...

https://github.com/open-api-spex/open_api_spex/blob/08df07c4ce93210f10583070ae3bdcd4430d9d30/lib/open_api_spex/controller_specs.ex#L368

Having `noapi_operation` return something else than `nil` will obviously test something, but will not test the equivalent behavihor to `operation :noapi, :false`.

WDYT?

Reply:
In this case, I suggest we change the test to have `noapi_operation` return `false`. imho at some point we should be explicit and only disable the specs for an action only with `false`.

Reply:
Hi Dimitri, as I said above ...

the macro code here ...

https://github.com/open-api-spex/open_api_spex/blob/08df07c4ce93210f10583070ae3bdcd4430d9d30/lib/open_api_spex/controller_specs.ex#L368

... generates a function that looks like this ...

`def noapi_operation(), do: nil`

... when you use the dsl like this ...

`operation :no_api, :false`

Are you sure you want this test to test ...

`def noapi_operation(), do: :false`

... which is as far as I can see it an undefined behaviour.